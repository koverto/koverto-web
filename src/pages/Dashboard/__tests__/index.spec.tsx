import { render } from "@testing-library/react"
import * as React from "react"
import { MemoryRouter } from "react-router-dom"
import { Dashboard } from ".."

describe("Dashboard", () => {
  it("matches the snapshot", () => {
    const { container } = render(
      <MemoryRouter>
        <Dashboard />
      </MemoryRouter>
    )

    expect(container).toMatchSnapshot()
  })
})
