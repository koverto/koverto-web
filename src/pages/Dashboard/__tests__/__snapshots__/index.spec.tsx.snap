// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard matches the snapshot 1`] = `
<div>
  <h1>
    Dashboard
  </h1>
</div>
`;
